test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -253,3 +254,124 @@ func schedulePods(config *testConfig) int32 {
 		time.Sleep(1 * time.Second)
 	}
 }
+
+func (nodeAffinity *nodeAffinity) mutateNode(numNodes int) []testutils.CountToStrategy {
```

Comment:
If this compiles then it means we're not using the interfaces idiomatically. 
https://play.golang.org/p/wf3V69v9N5